src/main/java/mate/academy/rickandmorty/config/BeanConfig.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.rickandmorty.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class BeanConfig {
+    @Bean
+    public ObjectMapper getObjectMapper() {
```

Comment:
```suggestion
    public ObjectMapper objectMapper() {
```

name of method is bean name in this case, so should be edited as bean name